src/main/java/uk/gov/hmcts/reform/pip/data/management/database/AzureTableService.java

```diff
@@ -0,0 +1,124 @@
+package uk.gov.hmcts.reform.pip.data.management.database;
+
+import com.azure.data.tables.TableClient;
+import com.azure.data.tables.models.ListEntitiesOptions;
+import com.azure.data.tables.models.TableEntity;
+import com.azure.data.tables.models.TableServiceException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.pip.data.management.config.PublicationConfiguration;
+import uk.gov.hmcts.reform.pip.data.management.errorhandling.exceptions.PublicationException;
+import uk.gov.hmcts.reform.pip.data.management.models.publication.Artefact;
+import uk.gov.hmcts.reform.pip.data.management.models.publication.ArtefactType;
+import uk.gov.hmcts.reform.pip.data.management.models.publication.Language;
+import uk.gov.hmcts.reform.pip.data.management.models.publication.Sensitivity;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ * This is the repository layer that deals with Publications in the Azure table service.
+ */
+@Component
+public class AzureTableService {
+
+    private final TableClient tableClient;
+
+    @Autowired
+    public AzureTableService(TableClient tableClient) {
```

Comment:
It can :) However I've looked into this previously and found it's generally better to go down the constructor over the field injection route (testing, less duplicate code). Happy to change it though